src/styles/page.scss

```diff
@@ -0,0 +1,43 @@
+.page {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+
+  .productNameFont {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $boldTextColor;
```

Comment:
The variable `$boldTextColor` is used but not defined. Ensure it is defined in your SCSS files to avoid compilation errors.